io.github.chris2511.xca.appdata.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>io.github.chris2511.xca</id>
+    <name>XCA</name>
+    <summary>X Certificate and Key management</summary>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>LicenseRef-proprietary=https://github.com/chris2511/xca/raw/master/COPYRIGHT</project_license>
```

Comment:
Hey @chris2511, is the license actually proprietary? Looks very similar to MIT at first glance but I wasn't sure.